src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets

```diff
@@ -105,6 +105,11 @@ Copyright (c) .NET Foundation. All rights reserved.
     </_GenerateRestoreGraphProjectEntryInputProperties>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <!--SDK Analysis level value should be updated in correspondance with the version of SDK this targeting-->
+    <SDKAnalysisLevel Condition=" '$(SDKAnalysisLevel)' == ''">9.1.00</SDKAnalysisLevel>
```

Comment:
We are not supposed to set this.

Somewhere in the .NET SDK, this value will be set, and we should only consume it.

Reply:
You are right. The dotnet sdk installer sets the value in `Microsoft.NETCoreSdk.BundledVersions.props` which is consumed early by msbuild.